<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filewidgetClass</class>
 <widget class="QWidget" name="filewidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>filewidget</string>
  </property>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Load ROM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="biosButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Bios</string>
   </property>
  </widget>
  <widget class="QLabel" name="romLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>No ROM is set</string>
   </property>
  </widget>
  <widget class="QLabel" name="biosLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>101</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>No Bios is set</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
